tests/integration/test_smoke.py

```diff
@@ -87,23 +86,26 @@
 # we've sent invalid params.
 ERROR_TESTS = {
     's3': {'ListObjects': {'Bucket': 'thisbucketdoesnotexistasdf'}},
+    'devicefarm': {'GetDevice': {'arn': 'arn:aws:devicefarm:REGION::device:f'}},
     'dynamodb': {'DescribeTable': {'TableName': 'unknowntablefoo'}},
     'sns': {'ConfirmSubscription': {'TopicArn': 'a', 'Token': 'b'}},
 }
 
-
-REGION_OVERRIDES = {
+DEFAULT_REGION = 'us-east-1'
+REGION = {
```

Comment:
What's the motivation for the rename?  I find `REGION_OVERRIDES` more clear than `REGION`.